### PR TITLE
fix#2278

### PR DESCRIPTION
**Pull request by ShawnSiao**
fix#22
result
78

### CODE REVIEW

**Comment by fengzhenbing** [comment]
This is  to fix  https://github.com/apache/incubator-shenyu/issues/2278

File: src/routes/Plugin/Common/Selector.js [hunk 1]
```diff
@@ -977,11 +987,159 @@ class AddModal extends Component {
...
+                                      {getFieldDecorator(fieldName, {
+                                        rules,
+                                        initialValue: defaultValue,
+                                      })(
+                                        <Input
+                                          addonBefore={<div style={{width: labelWidth}}>{item.label}</div>}
+                                          placeholder={placeholder}
+                                          key={fieldName}
+                                          onChange={e=> {
+                                            this.onDealChange(
```
Comment (fengzhenbing) [inline src/routes/Plugin/Common/Selector.js:1089]: Why did you put two parameter  `item  and item.label `
```
onDealChange = (value,label,item) => {
    if(label === item.label){
```
Is  label === item.label alway right ?

Comment (fengzhenbing) [reply src/routes/Plugin/Common/Selector.js:1089]: Maybe you can write more elegant

Comment (ShawnSiao) [reply src/routes/Plugin/Common/Selector.js]: Thanks,i have created a new pull request.

**Pull request closed without merging**